src/relax/op/op_common.h

```diff
@@ -39,27 +39,6 @@ bool EqualCheck(const PrimExpr& lhs, const PrimExpr& rhs);
 
 #define RELAX_REGISTER_OP RELAY_REGISTER_OP
```

Comment:
```suggestion
#define RELAX_REGISTER_OP(OpName) RELAY_REGISTER_OP("relax." OpName)
```
Currently when using `RELAX_REGISTER_OP("relax.add")` we should write `relax.`, but for other relax register macro such as `RELAX_REGISTER_BINARY_BROADCAST_OP("add")` we don't. 

Would it be better if we handle this in `RELAX_REGISTER_OP` and agree on that when we use macro starting with `RELAX_`, the op will be registered under `relax.` namespace defaultly?

Reply:
Great point! Updated.